std/stdio.d

```diff
@@ -3326,7 +3326,21 @@ void main()
         assert(dcharsOut == "foo");
     }
 
-/// Get the size of the file, ulong.max if file is not searchable, but still throws if an actual error occurs.
+/** Get the size of the file in bytes, ulong.max if file is not searchable, but still throws if an actual error occurs.
+Example:
+---
+import std.stdio, std.file;
+
+void main()
+{
+    auto file_handle = File(deleteme, "w");
```

Comment:
`deleteme` probably shouldn't be appearing in documented examples!

```d
// Purposefully not documented. Use at your own risk
@property string deleteme() @safe
```

Reply:
It is used in many places so I just tried to be consistent. I agree with you that it probably should not be used in examples but that means other examples need to be changed. If this pull request get merged I will change other examples to be consistent with this change